packages/sakuli-legacy/src/loader/legacy-project-properties.class.spec.ts

```diff
@@ -49,4 +49,26 @@ describe("LegacyProjectProperties", () => {
       expect(props.isUiOnly()).toBe(true);
     });
   });
+
+  describe("isBrowserReuse", () => {
+    it("should return true by default", () => {
+      expect(props.isReuseBrowser()).toBe(true);
+    });
+
+    it("should return browserReuse when set", () => {
+      props.reuseBrowser = true;
+      expect(props.isReuseBrowser()).toBe(true);
+    });
+
+    it("should return browserReuse if set even testsuiteBrowserReuse is set", () => {
+      props.reuseBrowser = false;
+      props.testsuiteReuseBrowser = true;
+      expect(props.isReuseBrowser()).toBe(false);
+    });
```

Comment:
Sakuli follows the general pattern that specific config overrules global config.
Therefore I'd expect `isReuseBrowser` to be `true` here.

Reply:
As discussed, we leave it like that. This is because `reuseBrowser` should only be used as cli argument with `sakuli run /path/to/suite --reuseBrowser=false`. We found this pattern for `browser` property as well.